tests/conftest.py

```diff
@@ -227,6 +231,87 @@ def assessment_object(group_object, page_object, campaign_object):
     )
 
 
+@pytest.fixture
+def multiple_campaign_object(campaign_object):
```

Comment:
This function doesn't appear to use the `campaign_object` input, but it does seem to use the undeclared `template_object` and `smtp_object`.

Reply:
Corrected what was being used in 0605cb6.